packages/bbui/src/Form/Core/TextArea.svelte

```diff
@@ -1,33 +1,38 @@
-<script>
+<script lang="ts">
   import "@spectrum-css/textfield/dist/index-vars.css"
   import { createEventDispatcher } from "svelte"
 
   export let value = ""
-  export let placeholder = null
+  export let placeholder: string | undefined = undefined
   export let disabled = false
   export let readonly = false
-  export let id = null
-  export let height = null
-  export let minHeight = null
+  export let id: string | undefined = undefined
+  export let height: number | string | undefined = undefined
+  export let minHeight: number | string | undefined = undefined
   export const getCaretPosition = () => ({
     start: textarea.selectionStart,
     end: textarea.selectionEnd,
   })
   export let align = null
 
   let focus = false
-  let textarea
+  let textarea: HTMLTextAreaElement
   const dispatch = createEventDispatcher()
-  const onChange = event => {
-    dispatch("change", event.target.value)
+  const onChange = (event: FocusEvent) => {
+    const target: HTMLTextAreaElement | null =
+      event.target as HTMLTextAreaElement
```

Comment:
I've found the typing around events to be pretty awful to work with. Target is not actually nullable - it will always exist, so I usually just forcefully cast this to be an element type.